addon/components/pie-chart.js

```diff
@@ -162,11 +162,17 @@ export default ChartComponent.extend(FloatingTooltipMixin,
   }),
 
   otherData: Ember.computed('sortedDataWithOther.[]', 'sortFunction', function() {
-    var _ref;
     var otherSlice = _.find(this.get('sortedDataWithOther'), function(d) {
       return d._otherItems;
     });
-    var otherItems = (_ref = otherSlice != null ? otherSlice._otherItems : void 0) != null ? _ref : [];
+
+    var otherItems;
+    if (otherSlice != null && otherSlice._otherItems != null) {
```

Comment:
Unfortunately `!=` is actually the intended behavior here. This is the result of compiling `otherSlice?` in coffescript, so it should also check for `undefined`